src/it/saxon/src/test/java/com/jcabi/saxon/SaxonSampleTest.java

```diff
@@ -91,7 +95,14 @@ public Void call() throws Exception {
                 );
                 return null;
             }
-        } .call();
+        };
+        final ExecutorService executorService = Executors.newFixedThreadPool(5);
+        for (int count = zerotime; count < fiftytimes; count = count + 1) {
+            executorService.submit(callable);
+        }
+        executorService.shutdown();
+        executorService.awaitTermination(tentimes, TimeUnit.SECONDS);
```

Comment:
@tmjee Hereinafter, `awaitTermination` returns `true` if all tasks are finished successfully in given time.
As I said in on of my comments, we should assert that  `awaitTermination` returns `true` and fail overwise.

```
MatcherAssert.assertThat(executorService.awaitTermination(tentimes, TimeUnit.SECONDS), Matchers.is(true));
```